enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/controller/api/ConfigurationController.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * PowerAuth Enrollment Server
+ * Copyright (C) 2022 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.app.onboardingserver.controller.api;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Map;
+
+/**
+ * Controller publishing configuration to inform the client app.
+ *
+ * @author Lubos Racansky, lubos.racansky@wultra.com
+ */
+@RestController
+@RequestMapping(value = "api/configuration")
+public class ConfigurationController {
+
+    @Value("${enrollment-server-onboarding.onboarding-process.enabled}")
+    private boolean onboardingEnabled;
+
+    @GetMapping
+    public Object fetchConfiguration() {
```

Comment:
Quite free structure to extend later on. Exposed at _onboarding-server_, may be aggregated by _mobile utility server_ (if needed), that's why the signature is not required.

`GET /api/configuration`
```json
{
  "onboarding": {
    "enabled": true
  }
}
```